### PR TITLE
homepage&setup

### PR DESCRIPTION
**Pull request by itsmebipul00**
Large no of files are due to me having to change my react from 18 to 17.0.2 and also install all required scripts to start mockbee
Files not to be review : backend

### CODE REVIEW

**Review by humzahasan — APPROVED** [review]
- Good use of reusability of components
- UX focus design with loader and error handling to minimize blank screen
- Good structuring of files

Good to merge!

File: src/Screens/HomeScreen/HomeScreen.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const HomeScreen = () => {
+	const {
+		categories,
+		categoriesLoading,
+		categoriesError,
+		fetchCategories,
+	} = useCategory()
+
+	console.log(categories, categoriesLoading, categoriesError)
```
Comment (iRohitGaur) [inline src/Screens/HomeScreen/HomeScreen.js:23]: suggestion: remove `console.log`

**Review by iRohitGaur — APPROVED** [review]
UI looks great!

Nice work. Good to merge 👍🏻

File: src/Screens/HomeScreen/HomeScreen.js (hunk 1 above)
Comment (itsmebipul00) [reply src/Screens/HomeScreen/HomeScreen.js:23]: Thankyou will note it from next time to remove console logs while doing PRs

**Pull request merged**